.github/workflows/lint.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - run: yarn --frozen-lockfile
 
       # v1 is broken, the given sha was the latest master (but we dont want to track master)
-      - uses: tinovyatkin/action-eslint@8b32772f075413f9192f43641e03d9bd0abae621
+      - uses: hallee/eslint-action@master
```

Comment:
do these in other PRs ideally. its unclear why this is needed.

Reply:
I already tried `hallee/*` and it failed miserably.

Reply:
Sorry what's the requested change here?
Don't change the lint.yml so other branches use prev eslint?